test/ctia/test_helpers/es.clj

```diff
@@ -226,3 +228,34 @@
                         (:docs.count entry))}))
          (into {})
          walk/keywordize-keys)))
+
+(defmacro for-each-es-version [msg versions clean & body]
```

Comment:
:style/indent should be 3

Also please correct the argv/docstring/attrmap order